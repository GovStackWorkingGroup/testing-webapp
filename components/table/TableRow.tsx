import React from 'react';

const TableRow = () => (
  <div className='product-table-row'>
    <div className='details-arrow'></div>
    <div className='product-table-sections'>
      <div>
        <p></p>
      </div>
      <div>
        <p></p>
      </div>
      <div>
        <p></p>
      </div>
      <div>
        <p></p>
      </div>
    </div>
  </div>
);

export default TableRow;
