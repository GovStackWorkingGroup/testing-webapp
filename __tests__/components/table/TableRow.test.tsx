import TableRow from '../../../components/table/TableRow';
import { render } from '../../test-utils/test-utils';

describe('Unit tests for TableRow component:', () => {
  it('should render the TableRow component and match snapshot', () => {
    const { container } = render(<TableRow />);

    expect(container).toMatchSnapshot();
  });
});
