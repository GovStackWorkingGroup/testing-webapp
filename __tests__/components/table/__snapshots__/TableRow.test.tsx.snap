// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit tests for TableRow component: should render the TableRow component and match snapshot 1`] = `
<div>
  <div
    class="product-table-row"
  >
    <div
      class="details-arrow"
    />
    <div
      class="product-table-sections"
    >
      <div>
        <p />
      </div>
      <div>
        <p />
      </div>
      <div>
        <p />
      </div>
      <div>
        <p />
      </div>
    </div>
  </div>
</div>
`;
